server/service/appconfig.go

```diff
@@ -71,9 +71,6 @@ func getAppConfigEndpoint(ctx context.Context, request interface{}, svc fleet.Se
 	// only admin can see smtp, sso, and host expiry settings
 	if vc.User.GlobalRole != nil && *vc.User.GlobalRole == fleet.RoleAdmin {
 		smtpSettings = config.SMTPSettings
-		if smtpSettings.SMTPPassword != "" {
-			smtpSettings.SMTPPassword = "********"
-		}
```

Comment:
We were hidding it, but now we decide to show it to global admins? Correct?

Reply:
No, we are hiding the whole smtp setting for all except global admins. That remains true. But the blanking of the password was moved to the service to be able to test it easily.